tools/pubsys/src/aws/publish_ami/mod.rs

```diff
@@ -120,28 +119,26 @@ pub(crate) async fn run(args: &Args, publish_args: &PublishArgs) -> Result<()> {
         }
     );
 
-    // Parse region names, adding endpoints from InfraConfig if specified
+    // Parse regions, adding endpoints from InfraConfig if specified
```

Comment:
Are we still parsing the region string? 
```suggestion
    // Adding endpoints from InfraConfig if specified
```